node/src/components/chain_synchronizer/error.rs

```diff
@@ -96,19 +86,6 @@ pub(crate) enum Error {
     #[error(transparent)]
     BlockExecution(#[from] BlockExecutionError),
 
-    #[error(
-        "joining with trusted hash before emergency restart not supported - find a more recent \
-         hash from after the restart. \
-         last emergency restart era: {last_emergency_restart_era}, \
-         trusted hash: {trusted_hash:?}, \
-         trusted block header: {trusted_block_header:?}"
-    )]
-    TryingToJoinBeforeLastEmergencyRestartEra {
-        last_emergency_restart_era: EraId,
-        trusted_hash: BlockHash,
-        trusted_block_header: Box<BlockHeader>,
-    },
```

Comment:
Will these errors be re-introduced with https://github.com/casper-network/casper-node/issues/3233 ?

Reply:
I'm not sure. They might not be necessary anymore, as we should simply bail if we detect that the trusted hash points to a block from an old protocol version.
Actually, I'm not sure if we're doing that - probably not, so maybe I should include that in this PR, too.